src/App.css

```diff
@@ -1,3 +1,62 @@
-h1 {
-  color: red;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: rgba(18, 118, 129, 1);
+  --secondary-color: rgba(16, 55, 92, 1);
+  --paragraph-color: rgb(0, 0, 0);
+  --white-color: rgba(255, 255, 255, 1);
+  --main-font: "Roboto", sans-serif;
+  --item-color: rgba(217, 214, 221, 0.5);
+  --pop-bg: rgb(51, 48, 48);
+}
+
+.calculator {
+  background-color: var(--item-color);
+  width: 340px;
+  margin: auto;
+}
+
+.calculator-display {
+  font-size: 2em;
+  text-align: right;
+  color: #fff;
+  background-color: #adada9;
+  padding: 1px;
+  margin-top: 20%;
+}
+
+.calculator-keypad .btn-color {
+  background: #e6771cb9;
+}
+
```

Comment:
- [ ] Please make sure to use the correct color for the operators' buttons background `#FFA600` to follow the wireframe design.